src/tool_help.c

```diff
@@ -850,6 +850,12 @@ static const struct helptxt helptext[] = {
   {"    --xattr",
    "Store metadata in extended file attributes",
    CURLHELP_MISC},
+  {"    --clobber",
+   "Overwrite files that already exist, even for options that normally don't",
+   CURLHELP_OUTPUT},
+  {"    --no-clobber",
+   "Never overwrite files that already exist, instead appending .1 - .100",
+   CURLHELP_OUTPUT},
```

Comment:
Only document the `--no-clobber` one, and note that this list is (should be) generated - as documented at the top of it.